uuid.go

```diff
@@ -85,6 +85,9 @@ func (src *UUID) AssignTo(dst interface{}) error {
 		case *string:
 			*v = encodeUUID(src.Bytes)
 			return nil
+		case *Text:
+			v.Set(encodeUUID(src.Bytes))
+			return nil
```

Comment:
Is this part necessary?

Reply:
Not really, I remove it. There is not need to use it in this way from a `pgtype.Text`